src/ducks/balance/Balance.jsx

```diff
@@ -30,7 +30,7 @@ import { getAccountInstitutionLabel } from 'ducks/account/helpers'
 import { ACCOUNT_DOCTYPE, GROUP_DOCTYPE, SETTINGS_DOCTYPE } from 'doctypes'
 import History from './History'
 import historyData from './history_data.json'
-import { getBalanceHistories } from './helpers'
+import { getBalanceHistories, generateVirtualGroups } from './helpers'
```

Comment:
Suggestion : `s/generate/build/` ?

generate gives me the impression that they will stay on the server I don't know why. build is shorter also :)